### PR TITLE
fixed the tf issue #74564

### PR DESCRIPTION
**Pull request by sanskarmodi8**
I looked into the issue and found out that after the evaluation we are getting a list of size 4 where only the last one is the accuracy of the export_model. So ignored all the other output to only get the required loss and accuracy

### CODE REVIEW

File: site/en/tutorials/keras/text_classification.ipynb [hunk 1]
```diff
@@ -861,7 +861,7 @@
         ")\n",
         "\n",
         "# Test it with `raw_test_ds`, which yields raw strings\n",
-        "loss, accuracy = export_model.evaluate(raw_test_ds)\n",
+        "_, _, loss, accuracy = export_model.evaluate(raw_test_ds)\n",
```
Comment (MarkDaoust) [inline site/en/tutorials/keras/text_classification.ipynb:864]:
```suggestion
        "metrics = export_model.evaluate(raw_test_ds, return_dict=True)\n",
```

File: site/en/tutorials/keras/text_classification.ipynb [hunk 2]
```diff
@@ -861,7 +861,7 @@
...
         "print(accuracy)"
```
Comment (MarkDaoust) [inline site/en/tutorials/keras/text_classification.ipynb:865]:
```suggestion
        "print(metrics)"
```

**Review by MarkDaoust — CHANGES_REQUESTED** [review]

**Review by MarkDaoust — APPROVED** [review]

**Comment by MarkDaoust** [comment]
Thanks for the fix! 

I actually always prefer `return_dict=True` because then the response labels each output. It makes it much clearer what's what.

**Comment by sanskarmodi8** [comment]
> Thanks for the fix! 
> 
> I actually always prefer `return_dict=True` because then the response labels each output. It makes it much clearer what's what.

Thank you, Mark! I'll keep that in mind and make sure to use return_dict=True moving forward.
It definitely helps in making the output clearer.

**Pull request merged**